Autopilot/PathManager/Inc/pathDatatypes.hpp

```diff
@@ -0,0 +1,40 @@
+
+#ifndef PATH_DATATYPES_HPP
+#define PATH_DATATYPES_HPP
+
+struct Telemetry_Waypoint_Data_t {
+    long double latitude;
+    long double longitude;
+    int altitude;
+    float turnRadius;
+    int waypointType; // 0 = Path follow, 1 = Orbit, 2 = Hold
```

Comment:
Cab we make this an enum for readability?

Reply:
lol this structure was created by dhruv, no credit to me. I just implemented it so that I had some parameters for the functions. @Pop0097

Reply:
Could create an enum here, but it'll add complexity to CommsWithTelemetry state. Will add it once everything is taking shape to prevent merge conflicts and all